### PR TITLE
New Noodle & python sprites, adds python vore

### PR DESCRIPTION
**Pull request by MeepleMuncher**
## About The Pull Request

Changes Noodle and python sprites to a new sprite I made, which hopefully looks better?? (Please tell me if you think it looks bad.)

Also gave Noodle and pythons the ability to vore, and the needed sprites and code to do so. (hopefully, I've been working on this for the whole day)

snekbigsheet

## Changelog

:cl:
add: added vorebellies and bump vore (safe) to pythons
imageadd: added new python (Noodle) sprites
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm [hunk 1]
```diff
@@ -91,6 +101,24 @@
...
+	B.mode_flags = DM_FLAG_THICKBELLY
+	B.digest_brute = 2
+	B.digest_burn = 2
+	B.digest_oxy = 1
+	B.digestchance = 0
+	B.absorbchance = 0
+	B.escapechance = 25
+	B.escape_stun = 5
+
 /mob/living/simple_mob/animal/passive/snake/python/noodle/Life()
```
Comment (ItsSelis) [inline code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:122]:
```suggestion
// CHOMPEdit Start
/mob/living/simple_mob/animal/passive/snake/init_vore()
	if(!voremob_loaded)
		return
	.=..()
	var/obj/belly/B = vore_selected
	B.name = "stomach"
	B.desc = "The snake coils its tail around you, pushing you to the ground and pinning you with its weight. It flicks its tongue at you, before pouncing onto your head, engulfing the upper half of your body with ease as it unhinges its jaw. With greedy swallows, it pulls you deeper, and deeper. The tight walls undulate rhythmically as the danger noodle rumbles contentedly at this new meal. The snake sends the last of you down with a deep swallow, hissing softly and closing its eyes as it enjoys its new meal, tucked away nicely under those beautiful, green scales."
	B.digest_mode = DM_HOLD
	B.item_digest_mode = IM_HOLD
	B.mode_flags = DM_FLAG_THICKBELLY
	B.digest_brute = 2
	B.digest_burn = 2
	B.digest_oxy = 1
	B.digestchance = 0
	B.absorbchance = 0
	B.escapechance = 25
	B.escape_stun = 5
// CHOMPEdit End

/mob/living/simple_mob/animal/passive/snake/python/noodle/Life()
```

Comment (ItsSelis) [reply code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:122]: Bigger sections of code changes that are not in modular_chomp have to be marked with CHOMPEdit Start/End sections.

File: code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm [hunk 2]
```diff
@@ -71,10 +71,20 @@
...
+	icon = 'modular_chomp/icons/mob/animal_ch.dmi' //CHOMPEdit - Newer, cooler sprites??
+
+	vore_active = 1  //CHOMPEdit - adding vore, about time :p
+	vore_capacity = 1
+	vore_default_mode = DM_HOLD
+	vore_icons = SA_ICON_LIVING | SA_ICON_REST
+	vore_escape_chance = 25
+	swallowTime = 50
+	vore_bump_chance = 10
+	vore_bump_emote = "slithers around and tries to devour"
```
Comment (ItsSelis) [inline code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:87]:
```suggestion
        // CHOMPEdit Start - New sprites & Added vore
        /*
        icon_state = "python"
	icon_living = "python"
	icon_dead = "python_dead"
	icon = 'icons/mob/snake_vr.dmi'
	*/
	icon_state = "snek"
	icon_living = "snek"
	icon_dead = "snek_dead"
	icon_rest = "snek_rest"
	icon = 'modular_chomp/icons/mob/animal_ch.dmi'

	vore_active = 1
	vore_capacity = 1
	vore_default_mode = DM_HOLD
	vore_icons = SA_ICON_LIVING | SA_ICON_REST
	vore_escape_chance = 25
	swallowTime = 50
	vore_bump_chance = 10
	vore_bump_emote = "slithers around and tries to devour"
	// CHOMPEdit End
```

Comment (ItsSelis) [reply code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:87]: Bigger sections of code changes that are not in modular_chomp have to be marked with CHOMPEdit Start/End sections.

**Review by ItsSelis — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm (hunk 2 above)
Comment (MeepleMuncher) [reply code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:87]: thank you!!

**Comment by MeepleMuncher** [comment]
thank!!

**Review by ItsSelis — APPROVED** [review]

**Pull request merged**